rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/model/stats/generateModel.groovy

```diff
@@ -0,0 +1,79 @@
+package org.jboss.windup.reporting.stats;
```

Comment:
Actually, I'd like to get it in. The `TechnologiesStatsModel.java` is generated by this.
We can trash it but then, refactoring it will be more tedious than just adjusting this script.